### PR TITLE
style added to the file

### PR DESCRIPTION
**Pull request by samuel776**
this is the style changes i have added to our file;

### CODE REVIEW

File: debug.log [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[0908/113221.336:ERROR:crash_report_database_win.cc(469)] failed to stat report
+[0908/113221.338:ERROR:crash_report_database_win.cc(469)] failed to stat report
+[0909/071124.476:ERROR:crash_report_database_win.cc(469)] failed to stat report
+[0909/071124.480:ERROR:crash_report_database_win.cc(469)] failed to stat report
```
Comment (ceelogre) [inline debug.log]: What is the use of this file?

File: index.css [hunk 2]
```diff
@@ -1,15 +1,4 @@
...
-h2, h3 {
-    display: inline;
-}
-
-.h2-heading {
-    text-transform: capitalize;
-    display: block;
+    background-color: burlywood;
+    margin-left: 2rem;
 }
```
Comment (ceelogre) [inline index.css]: You've removed everyone's style but the idea was to build from what others started.

**Review by ceelogre — CHANGES_REQUESTED** [review]

File: debug.log (hunk 1 above)
Comment (41hulk) [inline debug.log]: Same question as @ceelogre

**Review by 41hulk — COMMENTED** [review]
Reviewed

File: debug.log (hunk 1 above)
Comment (samuel776) [reply debug.log]: actually, this debug.log appeared while i was rebasing and i tried to fix it and falsely hoped it resolved and but now i made the changes and it is fixed

File: index.css (hunk 2 above)
Comment (samuel776) [reply index.css]: I wrongly accepted the changes and now  i fixed the changes and it is updated as you can see

**Pull request merged**